src/hooks/useMqtt.js

```diff
@@ -0,0 +1,64 @@
+import Paho from 'paho-mqtt'
+import { useEffect } from 'react'
+import { useState } from 'react'
+import { v4 as uuidv4 } from 'uuid'
+
+export default function useMqtt() {
+  const [client, setClient] = useState(null)
+  const [messages, setMessages] = useState([])
+  const [curRoom, setCurRoom] = useState(null)
+
+  useEffect(() => {
+    if (client) {
+      client.connect({
+        onSuccess: console.log('Connected'),
```

Comment:
Probably more relevant for dev purposes rather than prod env (Maybe delete it or connect user feedback)